### ISSUE TITLE
DistilBERT model training

### ISSUE THREAD

**Issue by prizma-12269269** [12296115/20265#4]
Configured and trained the DistilBERT sequence-classification model using GPU acceleration, AdamW optimisation, a learning rate of 2e-5, two training epochs, weight decay of 0.01, gradient clipping and a linear warmup/decay learning-rate schedule.
